Arrowgene.Ddon.Server/DdonServer.cs

```diff
@@ -88,7 +88,6 @@ public void Stop()
         {
             _server.Stop();
             _consumer.Dispose();
-            Database.Stop();
```

Comment:
I'm not sure i like this, i get why it's moved, so it doesn't get called twice, but both the login and game servers use the database

Reply:
Why not close it in server command too instead of DdonGameServer?

Reply:
Nevermind I misread. My bad. I shouldn't review things so late at night